### ISSUE TITLE
Get rid of `isToggleable` flag from `ButtonView`.

### ISSUE THREAD

**Issue by Mati365** [ckeditor/ckeditor5#16611]
## 📝 Provide a description of the improvement

It's follow up of this [PR](https://github.com/ckeditor/ckeditor5/pull/16573). In this PR we improved toggled state indication of menu items, we also improved support for attaching `aria-checked` and `aria-pressed` in generic button view class, but there is one small problem regarding DX. At this moment, we have to specify both `isToggleable: true` and set any `role` that marks button as `checkbox` in order to set these `aria` attributes. It's not necessary, we think. We decided to do removal of `isToggleable` in another PR because there are many places where `isToggleable` is used, and many places where `isToggleable` is not even set (but it should be) and removing it in the original PR would make it even bigger than it is. The `role` attribute should be used alone to decide if `aria-pressed` or `aria-checked` should be set on an item. 

In scope of this issue is:

1. Remove `isToggleable` flag from `ButtonView` (and check all classes that inherits from such class because they used to define not used `isToggleable` flag).
2. Check if every "truly" toggleable button in the project has proper `aria-pressed` (or any other checkbox-like) attribute set. 

Remember that `aria-pressed` and `aria-checked` attribute should be set on an element with `false` string value, even if it has `isOn=false` and has "checkable" `role` set. 

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by CKEditorBot** [comment]
There has been no activity on this issue for the past year. We've marked it as stale and will close it in 30 days. We understand it may still be relevant, so if you're interested in the solution, leave a comment or reaction under this issue.

**Comment by CKEditorBot** [comment]
We've closed your issue due to inactivity. We understand that the issue may still be relevant. If so, feel free to open a new one (and link this issue to it).
